### ISSUE TITLE
Возможность сменить акцентный цвет

### ISSUE THREAD

**Issue by MaxBQb** [0niel/university-app#253]
**Связан ли ваш запрос на функцию с проблемой? Пожалуйста, опишите.**
Не все люди видят белый на голубом/синем, я, например, не вижу текста на badge вроде лк/пр, числа в кружке текущего дня, кнопки входа и версии приложения.

**Опишите решение, которое вы хотели бы**
Добавьте возможность менять основной акцентный цвет, подойдёт даже просто выбор из вариантов.
Зелёный, оранжевый, тёмно-жёлтый, к примеру.

**Опишите альтернативы, которые вы рассматривали**
Можно просто акцентный цвет сделать сильно темнее, но тогда это повлияет и на тех, у кого моих проблем нет :)

**Дополнительный текст**
![image](https://user-images.githubusercontent.com/27343275/194754293-f16eacb9-c871-46be-819b-ee3fb4c24f1c.png)
Чем слабее зрение, тем сильнее сложности уидеть светлое на светлом, когда остальной фон тёмный.
Вроде бы даже в Material Dark просят не заливать большие области цветом.

**Comment by ValeryVerkhoturov** [comment]
> Не все люди видят белый на голубом/синем, я, например, не вижу текста на badge вроде лк/пр, числа в кружке текущего дня, кнопки входа и версии приложения.

Проблема связана с экраном Вашего устройства или физиологической особенностью?

**Comment by MaxBQb** [comment]
Второе :) 
Дальтонизм + миопия
Но думаю и одной миопия на достаточных минусах должно хватать для схожих жалоб.

**Comment by Dragonprod** [comment]
Настройки цветовой схемы Вашего устройства не решают проблему?

**Comment by MaxBQb** [comment]
Нет. Для подобных случаев максимум советы по Material Dark и контрасности фона/текста есть.
Ну и симуляции цветовых пространств, что вообще не меняет ситуации)
Кастомизация в любом случае прекрасная вещь, не думайте, что для одного человека фишку делаете :)

**Comment by 0niel** [comment]
@MaxBQb, привет. Решает ли белая тема проблему?

**Comment by MaxBQb** [comment]
Ну, я не находил возможность сменить тему. Если эта информация берётся из системных настроек.. так на Android 5.0 вообще выбора темы нет. Меня устраивает тёмная тема, просто цвета бы потвикать. А так, я бы предположил, что белая только усугубляет. Белый текст на таком же светлом условно сине-голубом фоне.. это недостаточно контрастное сочетание :) Особенно в ночи, когда всё что светится.. светится примерно одинаково.

**Comment by 0niel** [comment]
> Белый текст на таком же светлом условно сине-голубом фоне

Из PR #274:

![image](https://user-images.githubusercontent.com/51058739/215332704-11a329d1-83f7-4a34-8edf-ab2a3a3fe7fb.png)
![image](https://user-images.githubusercontent.com/51058739/215332669-eb7d97df-720e-42b5-8390-b17c8d9a3a9a.png)

Белая тема выглядит сейчас вот так.

**Comment by MaxBQb** [comment]
Если говорить только о контрасте, чёрный на белом это буквально идеально (как и белый на чёрном).
Хотя я предпочитаю тёмную тему. Обратите внимание, что незалитые (outlined) кнопки видны при любой теме, потому что фон контрастен к тексту, а кнопка "Настроить" будет видна по нормальному только в белой теме, потому что для тёмной фон останется тем же, а текст станет менее контрастным к этому фону (да, эта разница не для всех критична). Поэтому я и отметил, что допускаю как решения вроде, подкорректировать цвет, так и просто возможность его кастомизации.
